src/ai/backend/client/func/group.py

```diff
@@ -311,3 +313,101 @@ async def remove_users(
         }
         data = await api_session.get().Admin._query(query, variables)
         return data["modify_group"]
+
+    @api_function
+    @classmethod
+    async def get_container_registry_quota(cls, group_id: str) -> int:
+        """
+        Delete Quota Limit for the group's container registry.
+        Currently, only the HarborV2 registry is supported.
+
+        You need an admin privilege for this operation.
+        """
+        query = textwrap.dedent(
+            """\
+            query($id: String!) {
+                group_node(id: $id) {
+                    registry_quota
+                }
+            }
+        """
+        )
+
+        variables = {"id": base64(f"group_node:{group_id}")}
+        data = await api_session.get().Admin._query(query, variables)
```

Comment:
@fregataa Registry quoto READ operation can be executed even if the user is not an admin.
However, it seems that GQL queries in the current SDK can only be executed through `Admin`.
What do you think about adding the query to `User` as well?